src/auth/auth0-plugin.js

```diff
@@ -64,11 +64,12 @@ export const useAuth0 = ({
 
     async created() {
       this.auth0Client = await createAuth0Client({
-        ...pluginOptions,
         domain: pluginOptions.domain,
-        client_id: pluginOptions.clientId,
-        audience: pluginOptions.audience,
-        redirect_uri: redirectUri,
+        clientId: pluginOptions.clientId,
```

Comment:
I removed the spreading of `pluginOptions` above. We might need to consider bringing it back, but it wasn't clear why it was done in the sample.